ngraph/test/util/test_case.hpp

```diff
@@ -182,6 +182,23 @@ namespace ngraph
                 EXPECT_TRUE(res);
             }
 
+            void run_with_tolerance_as_fp(const float tolerance = 1.0e-5f)
+            {
+                m_engine.infer();
+                const auto res = m_engine.compare_results_with_tolerance_as_fp(tolerance);
```

Comment:
Looks like to check you can use cycle and `EXPECT_NEAR` from gtest instead of all these `compare_with_fp_tolerance` homemade methods.

Reply:
It is not true: `TestCaseEngine` class and its derived classes do not give access to calculation results.